addons/interact_menu/ACE_Settings.hpp

```diff
@@ -1,33 +1,4 @@
 class ACE_Settings {
-    class GVAR(alwaysUseCursorSelfInteraction) {
```

Comment:
All removed classes need to be re-added with `movedToSQF = 1;` (other entries can be removed).